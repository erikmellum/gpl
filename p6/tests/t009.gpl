string window_title = "p6 t009.gpl -- compare to t009.jpg";

textbox text1(x = 100, y = 100, text = 42, size = 0.1);
textbox text2(x = 200, y = 100, text = 42.2, size = 0.2);
textbox text3(x = 300, y = 100, text = "forty-two", size = 0.3);
