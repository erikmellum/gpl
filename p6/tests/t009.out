gpl.cpp::main()
  input file(tests/t009.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(true, file = results/t009.mypixels)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
game_object text1
  Textbox
  {
    animation_block = animation_block(0),
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    space = int(10),
    text = string("42"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(100),
    y = int(100)
  }
game_object text2
  Textbox
  {
    animation_block = animation_block(0),
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.2),
    space = int(10),
    text = string("42.2"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(200),
    y = int(100)
  }
game_object text3
  Textbox
  {
    animation_block = animation_block(0),
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.3),
    space = int(10),
    text = string("forty-two"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(10),
    x = int(300),
    y = int(100)
  }
string window_title "p6 t009.gpl -- compare to t009.jpg"
gpl.cpp::main() Calling window->initialize().
gpl.cpp::main() Passing control to window->main_loop().

Printing the symbol table from signal_handler()
game_object text1
  Textbox
  {
    animation_block = animation_block(0),
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(11),
    proximity = int(4),
    red = double(0),
    size = double(0.1),
    space = int(10),
    text = string("42"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(17),
    x = int(100),
    y = int(100)
  }
game_object text2
  Textbox
  {
    animation_block = animation_block(0),
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(21),
    proximity = int(4),
    red = double(0),
    size = double(0.2),
    space = int(10),
    text = string("42.2"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(60),
    x = int(200),
    y = int(100)
  }
game_object text3
  Textbox
  {
    animation_block = animation_block(0),
    blue = double(0),
    drawing_order = int(0),
    green = double(0),
    h = int(31),
    proximity = int(4),
    red = double(0),
    size = double(0.3),
    space = int(10),
    text = string("forty-two"),
    user_double = double(0),
    user_double2 = double(0),
    user_double3 = double(0),
    user_double4 = double(0),
    user_double5 = double(0),
    user_int = int(0),
    user_int2 = int(0),
    user_int3 = int(0),
    user_int4 = int(0),
    user_int5 = int(0),
    user_string = string(""),
    user_string2 = string(""),
    user_string3 = string(""),
    user_string4 = string(""),
    user_string5 = string(""),
    visible = int(1),
    w = int(194),
    x = int(300),
    y = int(100)
  }
string window_title "p6 t009.gpl -- compare to t009.jpg"
